docs/source/recipes/creating-project.md

```diff
@@ -7,15 +7,18 @@ myst:
     "keywords": "Volto, Plone, frontend, React, new, volto project, basic"
 ---
 
-# Creating a new Volto project
+# Create a Volto project without a backend
+
+This document shows how to create a Volto project with the frontend only when you have your own existing backend, such as Plone {term}`Classic UI`, {term}`Nick`, or {term}`Guillotina`.
 
 ```{seealso}
-This document shows how to create a Volto project with the frontend only.
 To create a full Plone project with both frontend and backend, see {doc}`plone:install/create-project` instead.
+
+To contribute to Volto, see {doc}`../contributing/developing-core`.
 ```
 
 For using Volto for a project—in other words, use Volto as a library—you should use Volto's project generator `@plone/generator-volto`.
-It's a boilerplate generator based in Yeoman that will provide you with the basic files and folder structure to bootstrap a Volto site.
+It's a boilerplate project generator based in Yeoman that will provide you with the basic files and folder structure to bootstrap a Volto site.
```

Comment:
@stevepiercy shouldn't it be project generator based on Yeoman instead of based in Yeoman?

Reply:
Yes, good catch.